src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,24 @@
+import classNames from 'classnames';
+
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
 // Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+
+type Prop = {
+  todo:Todo;
+};
+
+export const TodoInfo: React.FC<Prop> = ({ todo }) => (
+  <article
+    className={classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': todo.completed,
+      },
+    )}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+    { todo?.user && (<UserInfo user={todo.user} />)}
```

Comment:
```suggestion
    {todo?.user && <UserInfo user={todo.user} />}
```
or if component is big
```
  {todo?.user && (
    <UserInfo user={todo.user} />
  )}
```